### PR TITLE
Feature/ie 10 svg text rendering

### PR DESCRIPTION
**Pull request by apfelfabrik**
Fixes [`#2653`](https://www.openproject.org/issues/2653) and [`#2654`](https://www.openproject.org/issues/2654).

### CODE REVIEW

**Comment by apfelfabrik** [comment]
Merged current feature/rails3.

File: app/assets/javascripts/timelines.js [hunk 1]
```diff
@@ -4442,13 +4392,6 @@ Timeline = {
     var previousGroup = -1;
     var headerHeight = this.decoHeight();
 
-    // subtract 1px border if this is not firefox. in firefox, we need
-    // an additional pixel in the header. no idea where this comes from,
-    // it works fine in chrome and ie.
```
Comment (hschink) [inline app/assets/javascripts/timelines.js:4447]: And now it works in firefox too?

**Comment by hschink** [comment]
Tests run already for #649. So, as everything is fine, I merge this.

File: app/assets/javascripts/timelines.js [hunk 2]
```diff
@@ -4883,17 +4804,7 @@ Timeline = {
                 previousElement.outerHeight();
           } else {
 
-            // Reading decoHeight does not work equally well in
-            // SVG/VML and WebKit/Mozilla, so we need some additional
-            // adjustment. This is cumulative to the alterations for
-            // anti-aliasing below.
-
             previousEnd = timeline.decoHeight();
-            if (jQuery.browser.webkit) {
```
Comment (nkenneweg) [inline app/assets/javascripts/timelines.js:4892]: Removing this causes Bug [3072](https://www.openproject.org/work_packages/3072) and I would now rebuild the old behaviour (kind of). We might want to discuss browser-specific code some more.

**Pull request merged**